third_party/gson/BUILD.gn

```diff
@@ -0,0 +1,10 @@
+# Copyright 2016 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+
+java_prebuilt("gson_java") {
```

Comment:
Not sure how justified adding this lib for such a small class. It seems to me that adding own `toJson` and `fromJson` are more preferable. At least Chromium team goes this way.